packages/next/cli/next-dev.ts

```diff
@@ -90,9 +90,10 @@ const nextDev: cliCommand = (argv) => {
 
   startServer({ dir, dev: true, isNextDevCommand: true }, port, host)
     .then(async ({ app, actualPort }) => {
+      const { basePath } = await app.loadConfig()
```

Comment:
seemed to be the easiest way to access the `basePath`

Reply:
One of the main constraints of the Next.js dev bootup is that it does not block on anything that is not required to get the url onto the screen. In practice this means that there's no way to load the config before showing the url as the url has to be shown before any other work happens.

Reply:
@timneutkens So this means that the above wouldn't work or would block the boot?

Because, I tested it in several projects and it worked like a charm without any noticeable delays or problems 🤔

Reply:
> Why would loading and caching the parsed config be problematic before the server boots up?

It delays bootup on many projects as they have larger / more complex next.config.js configurations. Sometimes up to seconds.

Reply:
I see. Thanks for explaining. How about configuring the "basePath" then somewhere else where it could be read without big overhead? E.g. CRA is reading it from `package.json` using the "homepage" attribute, which I don't like so much as just having a "basePath" property in the "package.json". By making just another fallback it would be downwards compatible.

What do you think?

Reply:
Loading config from multiple sources is not preferable, it's just not possible to trade-off speed for a few characters being logged, especially when `basePath` is not being used by all apps.

Reply:
Ok. I understand your worries. Since you're much more into NextJS, I'd say you can just close this PR, if you think it's not worth it.

Let me one more time explain why I think it could be worth it:
- If you're using a `basePath` your assets and the `pages/index.ts` file are expected to be the entries to your app
- If you're using rewrite rules to you dev system for everything except for paths that are prefixed by the `basePath`, you'll find that opening the address printed without the `basePath` will be looking broken and this might confuse some devs.
- It increases the my manual overhead, having to click on the link and then going to the URL bar, writing the `basePath` on my own each time, and finally reloading the page

Like I said, if this is sufficient, I totally understand your decision to close the PR :+1:

Thank you for being so responsive to me 🙂